api/v1alpha1/backendtrafficpolicy_types.go

```diff
@@ -103,6 +104,15 @@ type BackendTrafficPolicySpec struct {
 	// +optional
 	// +notImplementedHide
 	Compression []*Compression `json:"compression,omitempty"`
+
+	// PerConnectionBufferLimitBytes Soft limit on size of the cluster’s connections read and write buffers.
+	// If unspecified, an implementation defined default is applied (32768 bytes).
+	// For example, 20Mi, 1Gi, 256Ki etc.
+	// Note: that when the suffix is not provided, the value is interpreted as bytes.
+	//
+	// +kubebuilder:validation:XValidation:rule="type(self) == string ? self.matches(r\"^[1-9]+[0-9]*([EPTGMK]i|[EPTGMk])?$\") : type(self) == int",message="PerConnectionBufferLimitBytes must be of the format \"^[1-9]+[0-9]*([EPTGMK]i|[EPTGMk])?$\""
+	// +optional
+	PerConnectionBufferLimitBytes *resource.Quantity `json:"perConnectionBufferLimitBytes,omitempty"`
```

Comment:
should we add a top level structure `Connection` like what we did in https://github.com/envoyproxy/gateway/blob/bd72474a1ff02b1e8c96efa6ea16f9b5ccb9b2e9/api/v1alpha1/connection_types.go#L14?

Reply:
IMO, we don't need the complexity of adding a parent type structure for this functionality.
+ First of all, this function is only used as a knob to supplement the configuration of the cluster's buffer.
+ Secondly, too much redundancy does not seem like a good thing (**less is more**). At least currently we have no plans to provide more other functions in BTP. In other words: If we need to add a `Connection` parent structure, maybe we should reconstruct the multiple types of BTP and divide them into `Connection`. (This may be implemented in the future)

Reply:
In this way, `ConnectionBufferLimit` maybe a simple naming.

Reply:
this looks good :)

Reply:
Done!